Lib/inspect.py

```diff
@@ -300,16 +300,16 @@ def isroutine(object):
 
 def isabstract(object):
     """Return true if the object is an abstract base class (ABC)."""
+    # TODO: RUSTPYTHON
```

Comment:
How about leaving the original code as commented-out and put this comments on line 305? So future contributor can notice what's changed in the source code.

Reply:
Sure! Fixed this.